### PR TITLE
Continuously libfuzz

### PR DESCRIPTION
**Pull request by killercup**

### CODE REVIEW

File: cli.rs [hunk 1]
```diff
@@ -203,12 +221,9 @@ fn run_honggfuzz(target: &str, timeout: Option<i32>) -> Result<(), Error> {
...
         ))?;
 
-    ensure!(
-        fuzzer_bin.success(),
-        "{} quit with code {}",
-        fuzzer,
-        fuzzer_bin
-    );
+    if !fuzzer_bin.success() {
+        Err(FuzzerQuit)?;
```
Comment (PaulGrandperrin) [inline cli.rs]: I'm very intrigued by this line.
Do people often use this notation instead of `return Err(FuzzerQuit);` ?

Comment (killercup) [reply cli.rs]: It's not the same—this converts the error!

File: cli.rs [hunk 2]
```diff
@@ -120,13 +120,22 @@ fn run() -> Result<(), Error> {
                 .iter()
                 .filter(|x| filter.as_ref().map(|f| x.contains(f)).unwrap_or(true));
 
-            if infinite {
-                for target in targets.cycle() {
-                    run(target)?;
+            'until_the_end_of_time: loop {
+                'all_the_fuzz: for target in targets.clone() {
```
Comment (PaulGrandperrin) [inline cli.rs]: I would suggest other labels like
`'infinite_cycle` and `'targets_pass`.

**Comment by PaulGrandperrin** [comment]
other than the small nits, LGTM

File: cli.rs (hunk 2 above)
Comment (killercup) [reply cli.rs]: How boring… but ooookay

File: cli.rs (hunk 1 above)
Comment (PaulGrandperrin) [reply cli.rs]: ah yes I forgot that, but then why not
```rust
return Err(FuzzerQuit).into();
```
I see that this is more verbose but I got used to having a mental representation of `?` as a way to say "we are not interested in explicitly handling the error here but we want the use the result". 
And here, we are explicitly creating this error and we are not interested in the non-existent `Ok()` case.

I understand this is more coherent in the sense that with this, all errors come from "?" so this is easy to skim the code and see those.

**Comment by killercup** [comment]
oh? did that get lost when we moved to the rust-fuzz orga?

File: cli.rs (hunk 1 above)
Comment (PaulGrandperrin) [reply cli.rs]: Anyway this is an interesting notation that I never saw before. I just need some time to wrap my head around it.

**Comment by PaulGrandperrin** [comment]
that weird, first I have no idea why i'm in the reviewers and you ain't.
and second, I can add you, but then when I click the Update button, you, and only you, disappears...
maybe @frewsxcv has an idea?

File: cli.rs (hunk 1 above)
Comment (killercup) [reply cli.rs]: Interesting, I don't see `?` as "let's not care about the error here", but more as a "[bubble](https://internals.rust-lang.org/t/bikeshed-a-consise-verb-for-the-operator/7289/29?u=killercup) up the error".

**Comment by killercup** [comment]
> and second, I can add you, but then when I click the Update button, you, and only you, disappears...

It's probably because I bothered @notriddle to many stupid questions about bors without contributing ;)

**Comment by PaulGrandperrin** [comment]
he banned you github-wide, that's tough!

**Comment by notriddle** [comment]
Luckily, I haven't had to ban anyone from bors itself yet, since there's no blocklist to add misbehaving users to. 😊

The reason killercup got removed from the reviewers list is that your list is getting synced with GitHub's list of admins. Change the dropdown box, next to the "Update" button, from Admin to either None (if you just want to maintain the list of reviewers by hand) or Push (if you just want everybody with push access to the repo to also be a reviewer).

Obviously, the "click here to add reviewer" thingy should not be present when synchronization is turned on like that, since the sync process writes over the top of any changes you make...

<small><i>I have, however, banned users from the forum. They were all spambots, probably.</i></small>

**Comment by frewsxcv** [comment]
@PaulGrandperrin @killercup I made both of you owners for the organization, so hopefully you shouldn't run into permissions problems again!

**Pull request merged**